modules/electromagnetics/doc/content/benchmarks/DipoleAntenna.md

```diff
@@ -0,0 +1,210 @@
+# Dipole Antenna Benchmark
+
+This document summarizes and describes the half-wave dipole antenna benchmark.
+In this scenario, a vertically-oriented antenna structure is excited by a 1 GHz
+signal in an infinite vacuum. Then, the subsequent transmitted radiation is
+absorbed by a first-order absorbing boundary condition (specifically, the
+[VectorEMRobinBC.md] object), positioned five wavelengths away from the antenna
+center-point.
+
+## Model Geometry and Antenna Specifications
+
+!style halign=left
+A half-wave dipole antenna has a geometry shown in [dipole-geometry-figure].
+
+!media dipole_geometry.png
+  style=width:50%;margin:auto;
+  id=dipole-geometry-figure
+  caption=Half-wave dipole antenna with oscillating voltage and current standing wave.
+
+The oscillating voltage between the two antenna conductors produces a current
+standing wave distribution along the length of the dipole structure. The antenna
+resonates (and transmits) when the applied voltage wave has a wavelength equal to
+two times the dipole length. Thus, the resonant frequency of a half-wave dipole
+is given by
+
+!equation
+f_r = \frac{c}{\lambda} = \frac{c}{2L}
+
+where $f_r$ is the resonant frequency, $c$ is the speed of light, and
+$L$ is the length of the half-wave dipole antenna. In this benchmark, a 1 GHz
+resonant antenna will be modeled, and the parameters for the geometry are shown
+in [dipole-geometry-parameters]. Note that the feed gap was chosen
+so that it would be *much* smaller then the overall length of the antenna,
+as an attempt to make the antenna geometry more ideal. The domain radius was
+chosen to be suitably far from the antenna surface to optimize transmission
+through the absorbing boundary condition. As mentioned in the [VectorEMRobinBC.md]
+documentation, the first-order condition used was designed for flat surfaces, not
+curved ones. Setting the boundary far enough from the origin increases the accuracy
+of this assumption for this scenario.
+
+!table id=dipole-geometry-parameters caption=Antenna geometry parameters for the dipole antenna verification benchmark.
+| Parameter (unit) | Value |
+| - | - |
+| Resonant frequency (GHz) | 1 |
+| Wavelength, $\lambda$ (m) | 0.3 |
+| Antenna length, $L$ (m) | 0.15 |
+| Antenna feed gap ($L / 20$) (m) | 0.015 |
+| Domain radius ($5 \lambda$) (m) | 1.5 |
+
+## Governing Equations and Boundary Conditions
+
+!style halign=left
+In this simulation, both the real and imaginary components of the electric field
+wave will be simulated separately as vector finite element variables (with the
+magnitude displayed in [dipole-results-figure]), but the frequency-domain electric
+field wave equation in general is given by:
+
+!equation
+\nabla \times \nabla \times \vec{E} - \mu_0 \varepsilon \omega^2 \vec{E} = 0
+
+where
+
+- $\vec{E}$ is the complex electric field vector in V/m,
+- $\mu_0$ is the vacuum magnetic permability of the medium in H/m,
```

Comment:
Oof copy and paste typo. Thanks

Reply:
Are there hardcoded numbers ? why a specific unit system?

Reply:
There might be defaults here and there for a constant with a specific unit, but I've attempted to keep things mathematically generic and not hardcode a constant so that a user/dev could change it as necessary. This is, of course, dependent in some cases on the derivations of things like BCs and assumptions in certain interface conditions. I've kept to the SI system there and in my benchmarks/V&V documentation (as is the case here) so these are noted specifically for reader reference.